scripts/examples.sh

```diff
@@ -1,51 +1,67 @@
 #!/bin/sh
 
 for f in internal/generator/examples/*.feature; do
-  dir=$(dirname $f)
-  base=$(basename $f)
+  dir=$(dirname "$f")
+  base=$(basename "$f")
+  name=$(basename "$f" .feature)
+  language="en" # The default language.
+
+  # Check if the file name contains a hint about a specific language.
+  case "$name" in
+    *.*)
+        language=$(echo "$name" | awk -F'.' '{print $NF}')
```

Comment:
I came up with this notation `<name>.<language(optional)>.<feature>` specifically to fix this script. I can also support it by the app. This should make the `-language` flag optional if the feature file name contains language information. Although I am not sure if this will it be helpful or misleading. What do you think @hedhyw ?